### PR TITLE
[4.0] CPanel Option in one column

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27490.

### Summary of Changes
Because CPanel  has unreadable fields in some windows sizes I changed the two column view to a one column view

### Testing Instructions

1. Open Global Configuration an see that all options displayed in two columns. 
2. Apply this patch
3. Open Global Configuration again and see, that the options displayed in one column. 

### Expected result

Global Configuration   test   Administration

### Actual result

Global Configuration   test   Administration(1)

### Documentation Changes Required

No

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 2bdc3af353d6fa087a63a37c8d981d26fd9ccc45<br /><br />PR does what it claims to do, so I give it a good test.

If the result is nice or not might be a matter of taste. At least it is better than before regarding readability of the drop down options.

I hope other people will give feedback, too.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27613">issues.joomla.org/tracker/joomla-cms/27613</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 2bdc3af353d6fa087a63a37c8d981d26fd9ccc45<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27613">issues.joomla.org/tracker/joomla-cms/27613</a>.</sub>

**Comment by Quy** [comment]
Yes, much better, however, on phone, the label needs to be above the input.



Maybe this can be addressed in #27558.

**Comment by astridx** [comment]
@Quy Your are right. I made a separate PR for this in case that it could not be addressed in #27558: See: https://github.com/joomla/joomla-cms/pull/27615

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27613">issues.joomla.org/tracker/joomla-cms/27613</a>.</sub>

**Comment by wilsonge** [comment]
Rather than this maybe change to `column-count-lg-2 column-count-md-1`

File: layouts/joomla/content/options_default.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
 		<p><?php echo $displayData->description; ?></p>
 	<?php endif; ?>
 	<?php $fieldsnames = explode(',', $displayData->fieldsname); ?>
-		<div class="column-count-lg-2">
+		<div class="column-count-lg-1">
```
Comment (ciar4n) [inline layouts/joomla/content/options_default.php]: You can probably just remove this class entirely.

**Comment by astridx** [comment]
> Rather than this maybe change to column-count-lg-2 column-count-md-1

`lg` is `≥992px`, right?
The problem exists with `1400px`:
https://github.com/joomla/joomla-cms/issues/27490#issuecomment-573522513

So I would like to leave it that way.

File: layouts/joomla/content/options_default.php (hunk 1 above)
Comment (astridx) [reply layouts/joomla/content/options_default.php]: You are certainly right. But this doesn't have a disadvantage, does it? And I also think that someone who doesn't have that much experience can more easily see in the code that it's 1 column.

Comment (Quy) [reply layouts/joomla/content/options_default.php]: Remove it if not needed. Less for the browser to parse.

Comment (Quy) [reply layouts/joomla/content/options_default.php]: Also to be consistent with the other configuration pages.

**Comment by astridx** [comment]
I have now deleted the class.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c057244e3490d130bcf6d07663bfea9051e596bc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27613">issues.joomla.org/tracker/joomla-cms/27613</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on c057244e3490d130bcf6d07663bfea9051e596bc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27613">issues.joomla.org/tracker/joomla-cms/27613</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**